index.css

```diff
@@ -0,0 +1,120 @@
+html {
+  font-size: 16px;
+  text-wrap: balance;
+}
+
+body {
+  font-family: "Martel Sans", serif;
+}
+
+h1 {
+  font-size: 3rem;
+  font-weight: 800;
+  margin: 3rem 0rem;
+  text-wrap: balance;
+}
+
+h2 {
+  font-size: 2.5rem;
+  font-weight: 800;
+  margin: 1.5rem 0rem;
+  text-wrap: balance;
+}
+
+h3 {
+  font-size: 2rem;
+  font-weight: 600;
+  margin: 1rem 0rem;
+  text-wrap: balance;
+}
+
+p {
+  font-size: 1.25rem;
+}
+
+li {
+  font-size: 1.125rem;
+  margin-bottom: 0.5rem;
+}
+
+.grid {
+  columns: 1fr;
+  display: grid;
+  padding: 1rem;
+  rows: auto;
+}
+
+.flex {
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+
+  /* Apply bottom margin if paragraph has siblings. */
+  p:not(:only-child) {
+    margin-bottom: 1rem;
+  }
+}
+
+/*  
+  css grid container
+    display grid
+      mobile
+        columns 1fr
+        auto rows
+        padding 16px
+
+      desktop
+        padding: 32px
+
+    inner flexbox & children
+      mobile gap: 48px
+*/
+
+/* Fluid font-sizing
+  fallback font size in case browser doesnt support
+  clamp rule to declare min, max, and preferred font size
+*/
+
+/* Martel Sans: CSS classes */
+
+.martel-sans-extralight {
+  font-family: "Martel Sans", serif;
+  font-weight: 200;
+  font-style: normal;
+}
+
+.martel-sans-light {
+  font-family: "Martel Sans", serif;
+  font-weight: 300;
+  font-style: normal;
+}
+
+.martel-sans-regular {
+  font-family: "Martel Sans", serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.martel-sans-semibold {
+  font-family: "Martel Sans", serif;
+  font-weight: 600;
+  font-style: normal;
+}
+
+.martel-sans-bold {
+  font-family: "Martel Sans", serif;
+  font-weight: 700;
+  font-style: normal;
+}
+
+.martel-sans-extrabold {
+  font-family: "Martel Sans", serif;
+  font-weight: 800;
+  font-style: normal;
+}
```

Comment:
These classes were just for reference. We can remove them since we can always get them back from Google Font's setup instructions.